ui/src/app.js

```diff
@@ -1243,7 +1244,7 @@ function formatFiat (amount, fractionDigits) {
     maximumFractionDigits: fractionDigits,
     minimumFractionDigits: fractionDigits
   })
-  return splitNumber(localized, jsLocaleCode) + ' ' + fiatCode
+  return splitNumber(localized, jsLocaleCode)
```

Comment:
A lot of other functions rely on formatFiat for the fiat code concat, it should not be removed.